tempesta_fw/Makefile

```diff
@@ -50,6 +50,7 @@ tempesta_fw-objs = \
 	sock_srv.o \
 	ss_skb.o \
 	stress.o \
+	tfw_tfwcfg.o \
```

Comment:
It's better to name the new module `vhost`, `location` is a part of virtual host logic.